### DISCUSSION TITLE
Airflow metadata

### DISCUSSION THREAD

**Discussion by anandajayam** [apache/airflow#34396] [General]
I have changed the airflow metadata to RDS postgresql,  after that task get fails. But all the table like dag, task instance ,user or getting updated, but task fails.

**Comment by Taragolis** [comment]
Would be nice if you provide a bit more information:
- Airflow version
- Deployment method
- Logs from worker/scheduler and other services where your error happen

> I have changed the airflow metadata to RDS postgresql 

Could a bit more clarify here, because this phrase could stand for multiple things
- You migrate from one type of DB (SQLite, MySQL, MsSQL) to Postgres?
- You migrate from self-managed postgres to RDS?
- You create new installation of airflow and use Postgres on RDS?

**Reply by anandajayam** [reply]
I install Airflow , after install i used postgres as Metadata , in the schema tables are created, ike ab_user , all the data are in table , but when i trigget the dags its getting fail , no data nlog also ,bcoz task itself fails , so no data captured . but if change the metadata to default sqlite its working fine. i mean dags sucess . Airflow version : Version: [v2.7.1](https://pypi.python.org/pypi/apache-airflow/2.7.1) . Postgres : 15 . in airflow.cfg file i added sql_connect: sql_alchemy_conn= postgresql+psycopg2://postgres:pass@localhost:port/DB and schema : airflow. i have not change any other setting .  when i create the dag its saved in table , but when i schedule or trigger dag , the task getting fails . kindly help me out.



ERROR - Marking run <DagRun dataset_consumes_1 @ 2023-09-17 06:35:21.336150+00:00: manual__2023-09-17T06:35:21.336150+00:00, state:running, queued_at: 2023-09-17 06:35:21.369310+00:00. externally triggered: True> failed

**Reply by hussein-awala** [reply]
> ERROR - Marking run <DagRun dataset_consumes_1 @ 2023-09-17 06:35:21.336150+00:00: manual__2023-09-17T06:35:21.336150+00:00, state:running, queued_at: 2023-09-17 06:35:21.369310+00:00. externally triggered: True> failed

Could you provide the worker log? (the full scheduler log if you are using LocalExecutor)

**Reply by Taragolis** [reply]
> ERROR - Marking run <DagRun dataset_consumes_1 @ 2023-09-17 06:35:21.336150+00:00: manual__2023-09-17T06:35:21.336150+00:00, state:running, queued_at: 2023-09-17 06:35:21.369310+00:00. externally triggered: True> failed

This is general log information about change DAG state and it just mean: “DagRun completed in state failed due to one of DAG leaf finished in failed state”.

Without additional log information someone who might try to help would just guess. So it would be really nice to have this information

In additional you might try to run DAG in Docker by use [sample docker compose](https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html#running-airflow-in-docker) sample if you familiar with docker stack, please note it is just sample and not production ready solution, it use Postgres by default
